test/plugin.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+    "plugins": [
+        "mocha"
```

Comment:
Doesn't this mean we want `eslint-plugin-mocha` as a dev dependency? It currently works because the plugin is installed via `test/fixture/eslint-plugin/package.json` and npm installs dependencies flat into `node_modules`.

Reply:
NPM install this plugin to `\test\fixture\eslint-plugin\node_modules\`. And it is the case that I check here. The plugin must not installed in the root directory. Only somewhere else.